docs/rules/name-property-casing.md

```diff
@@ -22,18 +22,10 @@ export default {
 }
 ```
 
-:+1: Examples of **correct** code for `camelCase`:
```

Comment:
Why did you remove this? Recommending something is one thing, but we should still have the ability to configure rule as we see fit.

Reply:
Not necessarily. For example, the `new-cap` and `camelcase` rules don't include the ability to enforce `snake_case`. Some might argue they should be able to configure variable and constructor casing however they like, but if there's no use case, it's not included as an option.

Similarly, there are benefits for PascalCase and kebab-case in the `name` property, but I don't believe any scenario exists where you'd want to use camelCase. It would simply break convention both in JavaScript (constructors are PascalCase) and HTML (custom elements are kebab-case).

Reply:
Okay, if it's aligned with the rule now then cool :)